repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -76,24 +76,32 @@ Now if we wanted to, we could change the Docker options that define the environm
 If we wanted to forward random ports instead of explicitly declaring the port mapping, we could replace the `--publish 8080:8080` option with `--publish-all` instead.
 
 == Manually Building a Repose Docker Image
-The latest version of the Dockerfiles used to build the *Repose Docker* images published on Docker Hub can be found at:
+The Dockerfile's used to build v{project-version} of the *Repose Docker* images published on Docker Hub are:
 
-- https://raw.githubusercontent.com/rackerlabs/repose/master/repose-aggregator/artifacts/docker/src/docker/ubuntu/Dockerfile[Debian/Ubuntu]
-- https://raw.githubusercontent.com/rackerlabs/repose/master/repose-aggregator/artifacts/docker/src/docker/centos/Dockerfile[RedHat/Centos]
++++ <details><summary>Ubuntu</summary><div> +++
+----
+include::../../../../artifacts/docker/src/docker/ubuntu/Dockerfile[]
+----
++++ </div></details> +++
+
++++ <details><summary>CentOS</summary><div> +++
+----
+include::../../../../artifacts/docker/src/docker/centos/Dockerfile[]
+----
++++ </div></details> +++
 
 To build a custom *Repose Docker* image, follow these steps:
 
 . Acquire a *Repose* Dockerfile.
-.. Clone/Fork the *Repose* project.
-.. cURL/Wget one of the files linked above.
 . *Optionally* Modify the Dockerfile.
-. Run the following command from the root of the *Repose* project to build an image based on the Dockerfile for Ubuntu that will provide *Repose* v8.7.0.2:
+. Run the following command from the root of the *Repose* project to build an image based on the Dockerfile copy/pasted from above.
 +
+[source,bash,subs="verbatim,attributes"]
 ----
-docker build                                             \ <1>
-   --build-arg REPOSE_VERSION=8.7.0.2                    \ <2>
-   --tag local/repose:v8.7.0.2-ubuntu                    \ <3>
-   ./repose-aggregator/artifacts/docker/src/docker/ubuntu/ <4>
+docker build                          \ <1>
+   --build-arg REPOSE_VERSION={project-version} \ <2>
+   --tag local/repose:v{project-version}-local  \ <3>
+   /dir/containing/dockerfile/          <4>
```

Comment:
Aw, the callouts alignment was lost with this change.

Reply:
It will realign when the `-SNAPSHOT` is not present.